### PR TITLE
Fix `test ros` on master

### PR DESCRIPTION
**Pull request by ad-daniel**
**Description**
Apply the same workflows to master branch as well, as done in develop in #3173

### CODE REVIEW

**Comment by ad-daniel** [comment]
It appears to be working, although I'm not sure why the previous version of the workflow (in master) restricts the key to only ubuntu 18.04 whereas in develop no distinction is made. I've changed it as it is in develop, but if somebody knows the reason that'd be better.

**Comment by stefaniapedrazzi** [comment]
This is because master and develop branches are not synchronized.
The restriction on the master branch has been introduced by @lukicdarkoo recently (master branch only) in  #3180.

File: .github/workflows/test_suite_linux.yml [hunk 1]
```diff
@@ -56,12 +56,12 @@ jobs:
         export ROS_DISTRO=${{ matrix.ROS_DISTRO }}
         xvfb-run --auto-servernum make distrib -j4
     - name: Prepare Webots Controller Deployment
-      if: ${{ github.event_name == 'schedule' && matrix.os == 'ubuntu-18.04' }}
+      if: ${{ github.event_name == 'schedule' }}
```
Comment (lukicdarkoo) [inline .github/workflows/test_suite_linux.yml:59]:
```suggestion
      if: ${{ github.event_name == 'schedule' && matrix.os == 'ubuntu-18.04' }}
```
Don't change these

**Review by lukicdarkoo — APPROVED** [review]
Thank you

Let's just wait for the tests

**Pull request merged**